cmd/launcher/options.go

```diff
@@ -91,7 +92,9 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		// deprecated options, kept for any kind of config file compatibility
 		_ = flagset.String("debug_log_file", "", "DEPRECATED")
 	)
+
 	flagset.Var(&flOsqueryFlags, "osquery_flag", "Flags to pass to osquery (possibly overriding Launcher defaults)")
+	flagset.Var(&flAutoloadedExtensions, "autoloaded_extension", "Filenames of extensions to autoload, must be in same directory as launcher")
```

Comment:
Should probably support full paths, or same directory semantics. Could leave it for future work? Could also not expose it up here